cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
On thinking about this, I think we can make some more usage of a sanity checker - I am going to refactor to run the sanity checker when an old version is detected and also after a migration. This provides some additional error case checking. Pinging @harshavardhana for any related inputs about this.